sdk/core/Azure.Core/src/GeoJson/GeoPosition.cs

```diff
@@ -6,7 +6,7 @@
 namespace Azure.Core.GeoJson
 {
     /// <summary>
-    ///
+    /// Represents a single spatial position with altitude, longitude and optional latitude.
```

Comment:
Nit: [Oxford comma](https://www.cnn.com/2017/03/15/health/oxford-comma-maine-court-case-trnd/index.html) 🙂.
```suggestion
    /// Represents a single spatial position with altitude, longitude, and optional latitude.
```